extensions/amp-apester-media/0.1/utils.js

```diff
@@ -177,3 +179,24 @@ export function registerEvent(eventName, callback, win, iframe, unlisteners) {
   });
   unlisteners.push(unlisten);
 }
+
+/**
+ * Generates a pixel url
+ * @param {string} publisherId
+ * @param {string} affiliateId
+ */
+export function generatePixelURL(publisherId, affiliateId) {
+  const qsObj = {
```

Comment:
@erwinmombay is this type safe for single pass compilation?

Reply:
since the keys are strings this should be OK 👍